index.html

```diff
@@ -53,211 +53,92 @@
-            build it that way. So let's take a look at how we can make our
-            websites accessible for all our users.
-            <br />
-            <small
-              >Download slides
-              <a
-                href="./slides/NewDevs-A11y.pdf"
-                target="_blank"
-                rel="noopener noreferrer"
-                >here</a
-              ></small
-            >
-          </p>
-        </article>
-        <article>
-          <h3>Animations</h3>
-          <p>
-            Movement conveys meaning. Through animations we can grab the users
-            attention and make our content more meaningful. We will look at what
-            we can (and should) animate, how CSS animations work and what things
-            we should be paying attention when adding movement to websites.
-            <br />
-            <small
-              >Download slides
-              <a
-                href="./slides/Animations.pdf"
-                target="_blank"
-                rel="noopener noreferrer"
-                >here</a
-              ></small
-            >
-          </p>
-        </article>
-        <article>
-          <h3>Resources, Learning, Getting help</h3>
-          <p>
-            There is too much internet, it’s easy to get lost and not know what
-            to search or learn. This talk aims at showing useful things we have
-            found when scrapping the internet for materials. It will also guide
-            you through how some of us approach learning and asking for help.
-            <br />
-            <small
-              >Download slides
-              <a
-                href="./slides/Resources_Learning_ Getting_Help.pdf"
-                target="_blank"
-                rel="noopener noreferrer"
-                >here</a
-              ></small
-            >
-          </p>
-        </article>
-        <article>
-          <h3>Panel: One year in tech - the newcomers retrospective</h3>
-          <p>
-            So how is working in tech? What are the most challenging things
-            within your first years as a developer? Why not ask some developers
-            who jumped into tech only a few years ago. In this panel, we will
-            reveal all the highs and lows as a junior developer.
-          </p>
-        </article>
-      </section>
-      <section>
-        <h2>Join us</h2>
         <p>
-          This is a truly unique possibility for true development newbies to get
-          in touch with frontend development – join us and become a new dev on
-          the block in no time! Don’t miss out and apply now until June 5 –
-          spaces are limited to 20 participants only!
-          <br />
-          The workshop will be in english and please bring your own notebook.
+          If you wish to attend our workshop, please register <a href="https://forms.gle/kGtsSiLookczkHRq9">here</a>.
```

Comment:
I'd be great to make this link more descriptive (a11y) 💜

Maybe we can reword it to: `If you wish to attend our workshop, please fill out our registration form` with "fill out our registration form" linked (or something similar)

Reply:
I was honestly waiting for your input here 😆 

I was at a talk lately where this particular thing was mentioned, but I couldn't remember.